src/features/link/index.js

```diff
@@ -66,49 +66,49 @@ export const LinkPage = function () {
-
-            <div className="title field">link do wypełniania</div>
-            <div className="field linkField">
-                <a href={fillLink} id="linkDisplay" type="text">
-                    {fillLink}
-                </a>
-                <button
-                    id="linkCopyButton"
-                    onClick={() => navigator.clipboard.writeText(fillLink)}
-                >
-                    kopiuj
-                </button>
-            </div>
-
-            <div className="title field">
-                Data końca ankiety w formacie RRRR-MM-DD.
-            </div>
-            <div className="field endDateField">
-                <span autoFocus id="endDateInput" type="text">
-                    {endDate}
-                </span>
-            </div>
-
-            <div className="stats_table">
-                <MaterialTable
-                    style={{ width: "80%" }}
-                    title="Statystyki ankiety"
-                    data={stats}
-                    columns={columns}
-                    options={{
-                        search: true,
-                        paging: false,
-                        filtering: true,
-                        exportButton: true,
-                    }}
-                />
+        <div className="wrapper">
+            <div className="survey_finished">
+                <div className="field_success">Twoja ankieta:</div>
+                <div className="field">
+                    <span autoFocus id="nameDisplay" type="text">
+                        {nameValue}
+                    </span>
+                </div>
+
+                <div className="field_link_name">Skopiuj link</div>
+                <div className="link_field">
+                    <a href={fillLink} className="link" id="link" type="text">
+                        {fillLink}
+                    </a>
+                    <button
+                        className="link_copy_button"
+                        id="linkCopyButton"
+                        onClick={() => navigator.clipboard.writeText(fillLink)}
+                    >
+                        📋
+                    </button>
+                </div>
+
+                <div className="stats_table">
+                    <MaterialTable
+                        style={{ width: "100%" }}
+                        title="Statystyki ankiety"
+                        data={stats}
+                        columns={columns}
+                        // options={{
+                        //     search: true,
+                        //     paging: false,
+                        //     filtering: true,
+                        //     exportButton: true,
+                        // }}
+                    />
+                </div>
+                <div className="field">Ankieta wygaśnie:</div>
+                <div className="field">
+                    <span autoFocus id="endDateInput" type="text">
+                        {endDate}
```

Comment:
Ta data wyświetla się w średnio czytelnej, moim zdaniem, formie:

> Ankieta wygaśnie:
> 2022-05-15T00:00:00.000Z

Można by to zmienić na ```{`${new Date(endDate)}`}```, wtedy by się wyświetlał tak:

> Ankieta wygaśnie:
> Sun May 15 2022 02:00:00 GMT+0200 (czas środkowoeuropejski letni)

co wydaje mi się trochę lepsze.


A najlepiej użyć jakiegoś frameworka do formatowania daty i wypisać to jakoś po ludzku np.:

> 15.05.2022 02:00:00